### PR TITLE
adding debug/proxy-info to get proxy istio version

### PR DESCRIPTION
**Pull request by ChenglongGao327**
adding debug/proxy-info to get proxy istio version

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

- [ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: pkg/proxy/proxyinfo.go [hunk 1]
```diff
@@ -19,45 +19,32 @@ import (
 	"encoding/json"
 	"fmt"
 
-	"istio.io/istio/pilot/pkg/xds"
```
Comment (hzxuzhonghu) [inline pkg/proxy/proxyinfo.go:22]: Great, this can decouple importing `istio.io/istio/pilot`

**Review by hzxuzhonghu — APPROVED** [review]

**Review by howardjohn — COMMENTED** [review]
doesn't istioctl ps already show this info?

**Comment by hzxuzhonghu** [comment]
I think it is to remove dependency on the pilot package. But `istioctl/pkg/writer/pilot/status.go` also depend on `xds.SyncStatus`.

**Review by howardjohn — CHANGES_REQUESTED** [review]
Please explain why we need this since proxy status already has it. Thanks!

**Comment by ChenglongGao327** [comment]
> Please explain why we need this since proxy status already has it. Thanks!

sorry, missing it , i think it decouple importing ```istio.io/istio/pilot```, did not notice that istioctl ps can get these.

**Pull request closed without merging**